simple/src/hello.c

```diff
@@ -42,18 +43,23 @@ int main(void)
   PCC-> PCCn[PCC_PORTC_INDEX] = PCC_PCCn_CGC_MASK; /* Enable clock to PORT C */
   PCC-> PCCn[PCC_PORTD_INDEX] = PCC_PCCn_CGC_MASK; /* Enable clock to PORT D */
                                /* Configure port C12 as GPIO input (BTN 0 [SW2] on EVB) */
-  PTC->PDDR &= ~(1<<PTC12);    /* Port C12: Data Direction= input (default) */
-  PORTC->PCR[12] = 0x00000110; /* Port C12: MUX = GPIO, input filter enabled */
+
+  GPIOC->PDDR &= ~(1<<BUTTON_PIN);    /* Port C12: Data Direction= input (default) */
+  PORTC->PCR[BUTTON_PIN] = 0x00000110; /* Port C12: MUX = GPIO, input filter enabled */
```

Comment:
hi thanks for the PR please rewrite ``` 0x00000110 ``` as a OR between PCR_MUX and PCR_PFE as detailed in cahpter 12 of the reference manual.